security.md

```diff
@@ -85,6 +86,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2021-3584"></a>CVE-2021-3584: Remote code execution through Sendmail configuration
+
+Sendmail location and arguments, available via Administer - Settings, both accept arbitrary strings and pass them into shell. By default, only Foreman super administrator can access settings.
+
+*Mitigation:* Verify the both settings and remove edit_settings permissions to all roles and users until fixed. Alternatively, create settings named sendmail_location and sendmail_arguments in `settings.yaml` file to override the UI and make the values read-only.
+
+* Affects Foreman veersion 1.15 and higher
```

Comment:
```suggestion
* Affects Foreman version 1.15 and higher
```

Reply:
Looks quite Dutch :-) veersion 🇳🇱

Reply:
The Dutch translation is `versie`. `veer` means feather or spring (the metal kind that you can compress and it'll take its old shape again) but there's no `veersion` ;)

Reply:
Typo is still here

Reply:
Damn, rebased.